azure-pipelines.yml

```diff
@@ -21,3 +21,6 @@ variables:
 
 jobs:
 - template: azure-pipelines/build.yml
+- template: azure-pipelines/richnav.yml
+  parameters:
+    windowsPool: Hosted Windows 2019 with VS2019
```

Comment:
Can we remove the parameter here since it only runs in one environment?

Reply:
Yes.